frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -24,13 +25,15 @@ end
 
 function filemanagerutil.abbreviate(path)
     if not path then return "" end
-    local home_dir_name = G_reader_settings:readSetting("home_dir_display_name")
-    if home_dir_name ~= nil then
+    if G_reader_settings:isTrue("shorten_home_dir") then
```

Comment:
Do we keep this off by default?

Reply:
I think it was off by default.
I guess most people have enabled it (well, I don't really know :), but as the setting name changed, they won't have it enabled. I don't think it's worth handling a migration.
So, dunno, may be it should indeed be on by default.

Reply:
Unsure of what to do here

Reply:
Isn't there an isNilOrTrue or something?

Reply:
http://koreader.rocks/doc/modules/luasettings.html#LuaSettings:nilOrTrue

Reply:
Solved